File: service/src/main/java/com/bnorm/barkeep/service/UserService.java
// Copyright 2016 (C) BNORM Software. All rights reserved.
package com.bnorm.barkeep.service;

import com.bnorm.barkeep.model.User;

public interface UserService {

  User getUser(long userId);

  User createUser(User user);

  User setUser(long userId, User user);
}

File: service/src/main/kotlin/com/bnorm/barkeep/service/UserService.kt
// Copyright 2017 (C) BNORM Software. All rights reserved.
package com.bnorm.barkeep.service

import com.bnorm.barkeep.model.User
import com.bnorm.barkeep.model.UserSpec

interface UserService {

  fun getUser(id: Long): User

  fun createUser(user: UserSpec): User

  fun setUser(id: Long, user: UserSpec): User
}
